stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -70,45 +89,48 @@ tree
     └── gc-ansible-cloud-123434.json
 ----
 
-
-== Running the upgrade playbook
+[discrete]
+= Running the upgrade playbook
 
 The following instructions describe how to run the upgrade playbook.
 
 .Procedure
-. Pull the docker image for "ansible-on-clouds-ops" container having the tag with the version which you want to upgrade to.
+. Pull the docker image for the `ansible-on-clouds-ops` container with a tag the same as the version of the foundation deployment.
 +
 [NOTE]
 =====
-The "ansible-on-clouds-ops" container image with the tag must match with the version which you want to upgrade to. For example, if your foundation deployment version is 2.2.0, pull "ansible-on-clouds-ops" image with tag 2.2.1 to upgrade to version 2.2.1.
+The `ansible-on-clouds-ops` container image tag must match the version of your foundation deployment. 
+For example, if your foundation deployment version is 2.2.1, pull the `ansible-on-clouds-ops` image with tag 2.2.1.
 =====
 +
 [source,bash]
 ----
 $ export IMAGE=quay.io/ansible-on-clouds/ansible-on-clouds-ops:2.2.1
 $ docker pull $IMAGE
 ----
-+
-. Running the upgrade playbook as a container.
-+
-.. Set variable `GOOGLE_APPLICATION_CREDENTIALS` to the google cloud credential Json file path.
+
+[discrete]
+== Running the upgrade playbook as a container
+
+.Procedure
+. Set the `GOOGLE_APPLICATION_CREDENTIALS` variable to the Google cloud credential JSON file path.
 +
 [source,bash]
 ----
-$ export GOOGLE_APPLICATION_CREDENTIALS=<path to the google credential Json file>
+$ export GOOGLE_APPLICATION_CREDENTIALS=<path to the Google credential JSON file>
 ----
 +
-.. Replace `<absolute path to secrets dir>` with your path to the service account credential Json file for google cloud and `<absolute path of extra vars dir>` with the path to your directory where you `extra_vars.yaml` file exists.
-+
-Replace `<google-cloud-project-id>` with the project id for your google account and `<deployment-name>` with your foundation deployment name.
-+
-.. Use the following command to run the playbook.
+. Replace `<absolute path to secrets dir>` with the path to the service account credential json file for Google cloud.
+. Replace `<absolute path of extra vars dir>` with the path to the directory where your `extra_vars.yaml` file exists.
+. Replace `<google-cloud-project-id>` with the project id for your Google account.
```

Comment:
`<Google-cloud-project-id>`. Caps G. It's a minor change. Below command has `<Google-cloud-project-id>`